tests/align/test_all_operators.sh

```diff
@@ -40,37 +41,40 @@ generate_ff_tensor view_embedding
 generate_ff_tensor max
 generate_ff_tensor min
 generate_ff_tensor linear
+generate_ff_tensor gather
 
 
 #create torch tensorss
-conda activate pytorch;
-generate_torch_tensor add;
-generate_torch_tensor concat;
-generate_torch_tensor conv2d;
-generate_torch_tensor cos;
-generate_torch_tensor embedding;
-generate_torch_tensor exp;
-generate_torch_tensor flat;
-generate_torch_tensor getitem;
-generate_torch_tensor identity;
-generate_torch_tensor multiply;
-generate_torch_tensor pool2d;
-generate_torch_tensor reducesum;
-generate_torch_tensor relu;
-generate_torch_tensor reshape;
-generate_torch_tensor scalar_add;
-generate_torch_tensor scalar_multiply;
-generate_torch_tensor scalar_sub;
-generate_torch_tensor scalar_truediv;
-generate_torch_tensor sigmoid;
-generate_torch_tensor sin;
-generate_torch_tensor subtract;
-generate_torch_tensor tanh;
-generate_torch_tensor transpose;
-generate_torch_tensor view_embedding;
-generate_torch_tensor max;
-generate_torch_tensor min;
-generate_torch_tensor linear;
+conda activate pytorch
+generate_torch_tensor add
```

Comment:
perhaps you can put the names of the operators in a array and replace these instructions with a for loop?